requirements/pip.txt

```diff
@@ -22,8 +22,6 @@ jsonfield==2.0.2
 requests==2.21.0
 requests-toolbelt==0.9.1
 slumber==0.7.1
-lxml==4.3.1
-defusedxml==0.5.0
```

Comment:
I think those are needed when building without using the docker images. We should test that people building without docker can still use their local instance before merging.

Reply:
Looks like we still need these packages https://github.com/rtfd/readthedocs.org/pull/3082
At least before https://github.com/rtfd/readthedocs.org/pull/5293/ is merged

Reply:
We shouldn't care about people building without Docker. It's unsupported.

Reply:
Have we ever officially documented that we don't support this? I think it's useful for development and much easier for users when they are getting started. Making users install and download docker images adds a huge dependency to people trying to get started.

Reply:
We only have discussed this, but I agree, It would make harder to start contributing.

Anyway, these libs were needed for api v1.

Reply:
We didn't document this, but we talked in different opportunities. Most (or all?) of the build changes are tested under the process that uses Docker, so we can't guarantee that local building produces the same output than building with Docker.